SPECS/qtdeclarative/qtdeclarative.spec

```diff
@@ -88,59 +91,139 @@ popd
-%dir %{_qt5_libdir}/cmake/Qt5Qml/
-%{_qt5_libdir}/cmake/Qt5Qml/Qt5Qml_*Factory.cmake
-
-%files static
-%{_qt5_libdir}/libQt5QmlDevTools.a
-%{_qt5_libdir}/libQt5QmlDevTools.prl
-%{_qt5_libdir}/libQt5PacketProtocol.a
-%{_qt5_libdir}/libQt5PacketProtocol.prl
-%{_qt5_libdir}/libQt5QmlDebug.a
-%{_qt5_libdir}/libQt5QmlDebug.prl
-
-%files examples
-%{_qt5_examplesdir}/
+%{_qt_archdatadir}/mkspecs/features/*.prf
+%{_qt_archdatadir}/mkspecs/modules/*.pri
+%{_qt_bindir}/qml*
+%{_qt_headerdir}/Qt*/
+%{_qt_libdir}/cmake/Qt6BuildInternals/StandaloneTests/QtDeclarativeTestsConfig.cmake
+%{_qt_libdir}/cmake/Qt6LabsFolderListModel/*.cmake
+%{_qt_libdir}/cmake/Qt6LabsQmlModels/*.cmake
+%{_qt_libdir}/cmake/Qt6LabsSettings/*.cmake
+%{_qt_libdir}/cmake/Qt6PacketProtocolPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6Qml/*.cmake*
+%{_qt_libdir}/cmake/Qt6Qml/*.cpp.in
+%{_qt_libdir}/cmake/Qt6Qml/*.qrc.in
+%{_qt_libdir}/cmake/Qt6Qml/QmlPlugins/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlCompiler/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlCore/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlDebugPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlDomPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlImportScanner/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlIntegration/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlLocalStorage/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlModels/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlToolingSettingsPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlTools/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlTypeRegistrarPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlWorkerScript/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlXmlListModel/*.cmake
+%{_qt_libdir}/libQt6LabsFolderListModel.so
+%{_qt_libdir}/libQt6LabsQmlModels.so
+%{_qt_libdir}/libQt6LabsSettings.so
+%{_qt_libdir}/libQt6Qml.so
+%{_qt_libdir}/libQt6QmlCompiler.so
+%{_qt_libdir}/libQt6QmlCore.so
+%{_qt_libdir}/libQt6QmlLocalStorage.so
+%{_qt_libdir}/libQt6QmlModels.so
+%{_qt_libdir}/libQt6QmlWorkerScript.so
+%{_qt_libdir}/libQt6QmlXmlListModel.so
+%{_qt_libdir}/pkgconfig/*.pc
+%{_qt_libdir}/qt6/metatypes/qt6*_metatypes.json
+%{_qt_libexecdir}/qmlcachegen
+%{_qt_libexecdir}/qmlimportscanner
+%{_qt_libexecdir}/qmltyperegistrar
+%dir %{_qt_libdir}/cmake/Qt6LabsFolderListModel
+%dir %{_qt_libdir}/cmake/Qt6LabsQmlModels
+%dir %{_qt_libdir}/cmake/Qt6LabsSettings
+%dir %{_qt_libdir}/cmake/Qt6PacketProtocolPrivate
+%dir %{_qt_libdir}/cmake/Qt6Qml
+%dir %{_qt_libdir}/cmake/Qt6Qml/QmlPlugins
+%dir %{_qt_libdir}/cmake/Qt6QmlCompiler
+%dir %{_qt_libdir}/cmake/Qt6QmlCore
+%dir %{_qt_libdir}/cmake/Qt6QmlDebugPrivate
+%dir %{_qt_libdir}/cmake/Qt6QmlDomPrivate
+%dir %{_qt_libdir}/cmake/Qt6QmlImportScanner
+%dir %{_qt_libdir}/cmake/Qt6QmlIntegration
+%dir %{_qt_libdir}/cmake/Qt6QmlLocalStorage
+%dir %{_qt_libdir}/cmake/Qt6QmlModels
+%dir %{_qt_libdir}/cmake/Qt6QmlToolingSettingsPrivate
+%dir %{_qt_libdir}/cmake/Qt6QmlTools
+%dir %{_qt_libdir}/cmake/Qt6QmlTypeRegistrarPrivate
+%dir %{_qt_libdir}/cmake/Qt6QmlWorkerScript
+%dir %{_qt_libdir}/cmake/Qt6QmlXmlListModel
+/usr/modules/LabsFolderListModel.json
```

Comment:
should /usr/modules be hardcoded?  (Here and in other SPEC files?)

Reply:
changed to use %{_prefix}/modules instead